model/db.go

```diff
@@ -76,6 +78,32 @@ CREATE TABLE IF NOT EXISTS merged_activist_attendance (
   CONSTRAINT merged_activist_attendance_ukey UNIQUE (original_activist_id, target_activist_id, event_id)
 )
 `)
+
+	db.MustExec(`
+CREATE TABLE IF NOT EXISTS working_groups (
+  id INTEGER PRIMARY KEY AUTO_INCREMENT,
+  name VARCHAR(60) NOT NULL,
+  type TINYINT(1) NOT NULL,
+  group_email VARCHAR(100) NOT NULL,
+  CONSTRAINT working_groups_name_ukey UNIQUE (name)
+)
+`)
+
+	db.MustExec(`
+CREATE TABLE IF NOT EXISTS working_group_members (
+  working_group_id INTEGER NOT NULL,
+  activist_id INTEGER NOT NULL,
+  -- True if the activist is the point person of the working group.
+  -- There should be only one point person per working group, but we
+  -- don't restrict that on the backend.
+  point_person TINYINT NOT NULL DEFAULT '0',
```

Comment:
What was the reason for moving the point person field back to the working_group_members? I felt it was much easier to enforce the "single point person per group" constraint this way.